src/ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -1,6 +1,6 @@
 """
 elemental_difference
-====================
+
```

Comment:
Hi @moe-ad, everything under operators/ is generated automatically using a template. We'll need to update the template for this to stay that way.

Reply:
See [here](https://github.com/ansys/pydpf-core/blob/b7f89f7015c15746b8b95c3a2d7f7ac855d91359/src/ansys/dpf/core/operators/operator.mustache#L3)